conanfile.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from conans import ConanFile, CMake
+from conans.tools import Git
+import os
+
+class ImguiSFML(ConanFile):
+    """ Imgui-SFML library conan file.
+
+    Options:
+
+    * shared: True or False
+      Whether to build this library shared or static
+
+    * fPIC: True or False
+      Whether or not to use -fPIC option while building
+
+    * imconfig: None or String
+      Use 'None' if you want this library to provide default
+      ImGui's imconfig.h file. Otherwise, provide a path to the
+      imconfig.h file you want to include in the build.
+
+    * imconfig_install_folder: None or String
+      Use 'None' if you want this build to include imconfig.h file,
+      otherwise use anything else.
+    """
+
+    name = 'ImGui-SFML'
+    version = '2.0.1'
+    description = 'ImGui binding for use with SFML'
+    topics = ('conan', 'sfml', 'imgui')
+    url = 'https://github.com/eliasdaler/imgui-sfml'
+    homepage = 'https://github.com/eliasdaler/imgui-sfml'
+    author = 'eliasdaler <eliasdaler@yandex.ru>'
+    build_requires = 'sfml/2.5.1@bincrafters/stable'
+    license = 'MIT'
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    options = {
+        'shared': [True, False],
+        'fPIC': [True, False],
+        'imconfig': 'ANY',
+        'imconfig_install_folder': 'ANY'
+    }
+    default_options = {
+        'shared': False,
+        'fPIC': True,
+        'imconfig': None,
+        'imconfig_install_folder': None
+    }
+    exports_sources = ['*.cpp', '*.h', 'CMakeLists.txt', 'cmake*']
+    exports = 'LICENSE.md'
+    _imgui_dir = 'imgui'
+
+    def source(self):
+        git = Git(folder=self._imgui_dir)
+        git.clone('https://github.com/ocornut/imgui.git', 'v1.69')
```

Comment:
Not very nice... This should be a parameter too, I think.

Reply:
Also, need to check that clone succeeded and report error if not

Reply:
Btw, don't really like that we need to do "git.clone" here... I think I'll go with subrepo approach soon. See #81

Reply:
> Also, need to check that clone succeeded and report error if not

Regarding the success of cloning: it emits errors automatically and stops the whole build.

> This should be a parameter too, I think.

Can you please elaborate on this ? I just see ImGui as a dependency, so hardcoding its version is a safe way to establish future builds.

> I think I'll go with subrepo approach soon.

Aha, I saw that, and since it's `v3.0`, I just decided to do it the old way. :slightly_smiling_face:
Second thought, is subrepo technically any different from current `git.clone` ?

Reply:
>  I just see ImGui as a dependency, so hardcoding its version is a safe way to establish future builds

Then let's go with the latest possible ImGui version that ImGui-SFML can handle (1.70).
And the problem with hardcoding it is that ImGui releases more frequently than ImGui-SFML, so people might want to use newer ImGui with ImGui-SFML, but unable to do so.

As for subrepo - yes, but it'll allow me to "pin" ImGui version and clone exactly needed one unless specified otherwise. This will allow Conan script to be updated less often. Let's leave "git.clone" as is for now, but add ImGui version parameter.